### ISSUE TITLE
Printing Lab Checkpoints isn't working

### ISSUE THREAD

**Issue by emaicus** [Submitty/Submitty#1510]
I couldn't find an existing issue for this.

The problem is because print functionality hasn't been ported from the old TA grading server.

**Comment by NicholasCiuica** [comment]
I tested the lab checkpoint printing functionality today and saw that it doesn't work on main. I made a checkpoint gradeable, marked some checkpoints for students in Section 1, then clicked "print" on their section, and got a printable page with no marks indicated. See below. Afterwards I also tried assigning myself as the grader for section 1 and that did not change anything.

<img width="463" height="233" alt="Image" src="https://github.com/user-attachments/assets/a9b1fd51-4d6f-4576-acd0-0a565139567f" />
<img width="956" height="194" alt="Image" src="https://github.com/user-attachments/assets/c68552d2-3058-4942-b2be-19f612f0c3f4" />

If this issue is reproduceable, is it worth it to try to fix it? I know during our meetings it was mentioned how this feature used to be used more, but now we're not sure how useful it is. And if it really is broken, who knows for how long it has been broken... has anyone else tested this in a while?

Also, when clicking "print" on the null section in a lab checkpoint, it returns a printable table but with no rows. If null students are omitted from the print version, then the print option should not be available for the null section.
